ddtrace/tracer/writer.go

```diff
@@ -183,15 +184,15 @@ func (h *logTraceWriter) encodeSpan(s *span) {
 	first := true
 	for k, v := range s.Meta {
 		if first {
-			h.buf.WriteByte('"')
 			first = false
 		} else {
-			h.buf.WriteString(`,"`)
+			h.buf.WriteString(`,`)
 		}
-		h.buf.WriteString(k)
-		h.buf.WriteString(`":"`)
-		h.buf.WriteString(v)
-		h.buf.WriteString(`"`)
+		m, _ := json.Marshal(k)
+		h.buf.Write(m)
+		h.buf.WriteString(":")
+		m, _ = json.Marshal(v)
```

Comment:
Let's not ignore any errors, ever (except maybe in tests). If it's not nil, just `log.Error`